src/main/java/com/geekschool/controllers/IndexController.java

```diff
@@ -0,0 +1,21 @@
+package com.geekschool.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping(value = "/")
+public class IndexController {
+    @GetMapping
+    static String getIndex() {
+        return "blog-home";
+    }
+
+    @GetMapping(value = "/elements")
```

Comment:
What is elements?

Reply:
This is link for page with another elements from this template.